src/coreComponents/common/logger/Logger.hpp

```diff
@@ -202,13 +398,61 @@
  */
 #define GEOS_INFO( msg ) LVARRAY_INFO( msg )
 
+/**
+ * @brief Abort execution if @p lhs @p OP @p rhs.
+ * @param lhs The left side of the operation.
+ * @param OP The operation to apply.
+ * @param NOP The opposite of @p OP, used in the message.
+ * @param rhs The right side of the operation.
+ * @param msg The message to diplay.
+ */
+#define GEOS_ERROR_IF_OP_MSG( lhs, OP, NOP, rhs, msg ) \
+  GEOS_ERROR_IF_IMPL( lhs OP rhs, \
+                      msg << "\n" << \
+                      "Expected " << #lhs << " " << #NOP << " " << #rhs << "\n" << \
+                      "  " << #lhs << " = " << lhs << "\n" << \
+                      "  " << #rhs << " = " << rhs << "\n" )
+
+/**
+ * @brief Raise a hard error if two values are equal.
+ * @param lhs expression to be evaluated and used as left-hand side in comparison
+ * @param rhs expression to be evaluated and used as right-hand side in comparison
+ * @param msg a message to log (any expression that can be stream inserted)
+ */
+#define LOG_GEOS_ERROR_IF_EQ_MSG( lhs, rhs, msg ) GEOS_ERROR_IF_OP_MSG( lhs, ==, !=, rhs, msg )
```

Comment:
```suggestion
#define GEOS_LOG_ERROR_IF_EQ_MSG( lhs, rhs, msg ) GEOS_ERROR_IF_OP_MSG( lhs, ==, !=, rhs, msg )
```

These macros mostly start with `GEOS`

Reply:
Yes indeed, I made the necessary changes for the macros that started with LOG_GEOS instead of GEOS_LOG.